src/Uno.UI/UI/Xaml/Controls/Border/Border.wasm.cs

```diff
@@ -7,13 +7,21 @@
 using Uno.Disposables;
 using Windows.UI.Xaml.Media;
 using Uno.UI;
+using Windows.UI.Xaml.Shapes;
 
 namespace Windows.UI.Xaml.Controls
 {
 	public partial class Border
 	{
+		private readonly BorderLayerRenderer _borderRenderer;
+
 		public Border()
 		{
+			_borderRenderer = new BorderLayerRenderer(this);
+
+			Loaded += (s, e) => UpdateBorder();
+			Unloaded += (s, e) => _borderRenderer.Clear();
+			LayoutUpdated += (s, e) => UpdateBorder();
```

Comment:
Isn't this going to be done too often? @carldebilly

Reply:
Technically it should not be a problem, as the `UpdateLayer` will short-circuit if `Border` nor `Background` properties are changed. I actually done it this way so that we could later refactor this more easily as not the exact same "initialization" sequence can be used on all targets

Reply:
Eventually the LayoutUpdated event will be raised only when the layout slot effectively changes, but that's not the case right now.

I agree it whould be refactored on all targets at once.